frontend/packages/employee-frontend/src/api/unit.ts

```diff
@@ -32,7 +33,31 @@ function convertUnitJson(unit: JsonOf<Unit>): Unit {
   return {
     ...unit,
     openingDate: unit.openingDate ? LocalDate.parseIso(unit.openingDate) : null,
-    closingDate: unit.closingDate ? LocalDate.parseIso(unit.closingDate) : null
+    closingDate: unit.closingDate ? LocalDate.parseIso(unit.closingDate) : null,
+    daycareApplyPeriod: unit.daycareApplyPeriod
+      ? {
```

Comment:
It would be nicer to have a `parseOpenEndedPeriod` function and just use it here a couple of times

Reply:
I suggest we fix this as part of the larger daterange refactoring, for example as a method in a new typescript class Period

Reply:
OK, works for me